include/uv.h

```diff
@@ -1313,6 +1313,8 @@ enum {
 UV_EXTERN int uv_thread_getpriority(uv_thread_t tid, int* priority);
 UV_EXTERN int uv_thread_setpriority(uv_thread_t tid, int priority);
 
+UV_EXTERN int uv_set_threadpool_size(unsigned int number);
+
```

Comment:
Adding a `uv_get_threadpool_size()` would be a good idea, I think.

Reply:
That could be useful as well